src/mobile/apps/artwork/components/related_artworks/test/display_related_works.coffee

```diff
@@ -8,7 +8,7 @@ Artworks = require '../../../../../collections/artworks'
 
 describe 'Render Related Artworks', ->
 
-  xdescribe 'displayRelatedWorks', ->
+  describe 'displayRelatedWorks', ->
```

Comment:
I'm not sure why these were disabled but they appear to pass and more specs are better.